openedx/core/djangoapps/lang_pref/tests/test_middleware.py

```diff
@@ -160,8 +151,6 @@ def test_preference_cookie_overrides_browser(
         else:
             assert accept_lang_result == accept_lang_out
 
-        assert self.request.session.get(LANGUAGE_SESSION_KEY) == lang_session_out
```

Comment:
Does a replacement assertion make sense?

Reply:
The use case of this test is already satisfied, so we can skip the replacement assertion.